include/labwc.h

```diff
@@ -345,6 +345,7 @@ struct server {
 	struct osd_state {
 		struct view *cycle_view;
 		bool preview_was_enabled;
+		bool direction;
```

Comment:
Suggest a better word than `direction` - for example `forwards`.
Although given how we use it, I think it's easier to read the enum anyway, so prefer to keep that.

Reply:
I'm not hung up on the name, I picked direction because that's what it controls, forwards/backwards.